jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,21 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # JUL implementation: https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html
+  local allowed_log_levels=("ALL" "CONFIG" "FINE" "FINER" "FINEST" "INFO" "OFF" "SEVERE" "WARNING")
+
+  if [[ ! "${allowed_log_levels[@]}" =~ "${LOG_LEVEL}" ]]; then
+    log_warning "Log Level ${LOG_LEVEL} is not allowed, the allowed levels are ${allowed_log_levels[@]}"
```

Comment:
I think not all items from array will be printed using "@". use "*" instead and check for all values on logs.

Reply:
fixed